src/js/view/floating/floating-controller.ts

```diff
@@ -171,6 +172,11 @@ export default class FloatingController {
                     });
                 });
             }
+            setTimeout(() => {
+                this.transitionFloating(false);
+                // Perform resize and trigger "float" event responsively to prevent layout thrashing
+                this._model.trigger('forceResponsiveListener', {});
```

Comment:
This avoids triggering the response listener until the CLS avoiding transition is done.